### ISSUE TITLE
Use of unassigned variable 'key'PHP(PHP0412) (php7.4 arrow function)

### ISSUE THREAD

**Issue by flexchar** [DEVSENSE/phptools-docs#57]
Hi,

I am trying out the extension again I having invalid error `Use of unassigned variable 'key'PHP(PHP0412)` from this snippet:

![image](https://user-images.githubusercontent.com/9198896/80946878-b87f2400-8df7-11ea-88fc-0efc0e1a87cd.png)

Extension:
```
Name: PHP Tools for VS Code
Id: devsense.phptools-vscode
Description: Complete PHP language development integration. Fast. Easy. Fully featured.
Version: 1.0.4608
Publisher: DEVSENSE
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=DEVSENSE.phptools-vscode
```

**Comment by jakubmisek** [comment]
Thank you for the issue, we'll try to fix it within the next update.

**Comment by jakubmisek** [comment]
thank you for the issue. This should be fixed in the recent version `1.0.4698`
